shaders/src/main/glsl/samples/320es/stable_triangle.frag

```diff
@@ -22,20 +22,34 @@ precision highp int;
 layout(location = 0) out vec4 _GLF_color;
 uniform vec2 resolution;
 
+
+/*
+This shader rasterizes a triangle in perhaps the most
+inefficent way possible apart from raytracing, by 
+checking for each pixel whether we're inside a triangle.
+
+The point in triangle code was based on 

```

Comment:
"point in triangle" -> "pointInTriangle"

Reply:
Done